md2html.sh

```diff
@@ -16,7 +16,11 @@ while getopts "hl" opt; do
 done
 
 for file in "$@"; do
-	pandoc -f markdown_github "${file}" -t asciidoc -o "${file%.md}.tmp.adoc"
+        if grep -qi 'Red Hat Enterprise Linux release 9' /etc/redhat-release; then
```

Comment:
Please, don't do this.
First, checking for a specific release is very fragile. You don't have any guarantee that the OS the script will run on will be named exactly like that and contain/not contain the package that you need. It is much better to check for the actual thing that you need. i.e. pandoc or kramdoc. One example (for deciding between asciidoc and asciidoctor) can be found here: https://github.com/rear/rear/blob/6a3d0b4d5e73c69a62ce0bd209b2b38ffb462569/doc/Makefile#L10
Second, even in cases when you need to determine the OS version and release, there are much more structured approaches to that than searching for a string in a file. See os-release(5).

Reply:
Thanks a lot, I was feeling that reading strings in the file is bad. Thanks for the explanation, I've tried another thing now.